docs/admin/admin-ui/dashboard.md

```diff
@@ -0,0 +1,41 @@
+# Dashboard
+
```

Comment:
Consider giving an example link for the dashboard so that the user knows what it looks like.

Reply:
Provided a dashboard image for illustration.